### DISCUSSION TITLE
AList 桌面版的MacOS最低版本是多少？

### DISCUSSION THREAD

**Discussion by zmd99** [AlistGo/alist#3756] [Q&A]
看刚推出了桌面版想安装，下载了安装不了可能是系统版本低了，我现在用的是macOS Sierra（10.12.6)，最低什么版本可以运行桌面版？

**Comment by xhofe** [comment]
这取决于tauri的支持度：https://tauri.app/zh-cn/v1/references/webview-versions#webkit-macos-ios--linux

理论上10.12.6是支持的，安装不了报什么错吗
